PyU4V/provisioning.py

```diff
@@ -1859,9 +1870,13 @@ def add_new_volume_to_storage_group(
 
         add_volume_param.update({'volumeAttributes': [volume_attributes]})
 
+        if enable_mobility_id:
+            add_volume_param.update({'enable_mobility_id': enable_mobility_id})
```

Comment:
I take it that the REST call happily takes True boolean as well as 'true' string?